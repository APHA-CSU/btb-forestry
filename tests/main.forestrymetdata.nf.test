nextflow_process {

    name "Test Process forestryMetdata"
    script "main.nf"
    process "FORESTRY_META_DATA"

    test("Should generate metadata") {

        when {
            params {
                today = "01Jan25"
            }
            process {
                """
                input[0] = [ true ]
                input[1] = params.today
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
