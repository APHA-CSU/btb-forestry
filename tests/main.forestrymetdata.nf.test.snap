{
    "Should generate metadata": {
        "content": [
            {
                "0": [
                    "metadata.json:md5,fde07aacbc6baba587808dc635f4d116"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.6"
        },
        "timestamp": "2025-08-29T13:51:40.146442069"
    }
}